<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdentityWindow</class>
 <widget class="QWidget" name="IdentityWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>p2kontakt: New Identity</string>
  </property>
  <widget class="QLineEdit" name="input_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_email">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>E-Mail Address</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttons_confirm">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>110</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="input_date">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>110</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_never">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Never Expire</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons_confirm</sender>
   <signal>rejected()</signal>
   <receiver>IdentityWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
